ingress/controllers/gce/backends/backends.go

```diff
@@ -201,18 +197,31 @@ func (b *Backends) List() ([]interface{}, error) {
 	return interList, nil
 }
 
+func getBackendsForIGs(igs []*compute.InstanceGroup) []*compute.Backend {
```

Comment:
just curious. If not all ubernetes clusters are on gce, how do you handle it?   Or just does not support this case? 
Can GCE L7 even support compute backends which are not on gce?